src/Angor/Client/Pages/View.razor

```diff
@@ -315,16 +315,9 @@ else
                 }
                 <button class="btn btn-border mb-3" data-cy="INVEST_BUTTON"
                         @onclick="InvestInProject"
-                        disabled="@(timeLeft <= 0 ? "disabled" : null)"
-                        title="@(timeLeft <= 0 ? "The investing period is over" : null)">
-                    @if (timeLeft > 0)
-                    {
-                        <text>Invest Now</text>
-                    }
-                    else
-                    {
-                        <text>The investing period is over</text>
-                    }
+                        disabled="@(project == null || !CanInvest(project))"
+                        title="@(CanInvest(project) ? null : "Investment is not allowed for this project.")">
+                    Invest Now
```

Comment:
no I prefer how it was before just put text there instead of a button it is nicer I think